.github/workflows/ci.yml

```
@@ -132,18 +148,26 @@ jobs:
       - name: Install nextest
         uses: taiki-e/install-action@nextest
 
+      - name: Install cargo-llvm-cov
+        uses: taiki-e/install-action@cargo-llvm-cov
+
       - name: Test
         uses: actions-rs/cargo@v1
         with:
-          command: nextest
-          args: run --workspace --exclude matrix-sdk-integration-testing
+          command: llvm-cov
+          args: nextest --lcov --output-path lcov.info run --workspace --exclude matrix-sdk-integration-testing
```

```suggestion
          args: --workspace --exclude matrix-sdk-integration-testing nextest --lcov --output-path lcov.info
```
`nextest` is the command so you don't need `run`.
There's also a weird bug (https://github.com/taiki-e/cargo-llvm-cov/issues/151) where the options that are forwarded to nextest need to be before the `nextest` command